yarn.lock

```diff
@@ -3862,6 +3862,11 @@ cloneable-readable@^1.0.0:
     process-nextick-args "^2.0.0"
     readable-stream "^2.3.5"
 
+clones@^1.2.0:
```

Comment:
Why did the lockfile change even though you didn't change any package.json?

Reply:
There was a linting error saying that shared/babel-ast-utils/ was missing a dependency. I added it but then saw that something was merged back into master probably solving same problem. I will try to revert yarn.lock.